src/towncrier/test/test_project.py

```diff
@@ -36,3 +37,24 @@ def test_tuple(self):
 
         version = get_version(temp, "mytestproja")
         self.assertEqual(version, "1.3.12")
+
+
+class InvocationTests(TestCase):
+    def test_dash_m(self):
+        """
+        `python -m towncrier` works.
```

Comment:
"works" is one of those non-words called out by https://jml.io/pages/test-docstrings.html so let's say what working means

```suggestion
        `python -m towncrier` invokes the main entrypoint.
```